### ISSUE TITLE
`01019_alter_materialized_view_consistent` failed

### ISSUE THREAD

**Issue by serxa** [ClickHouse/ClickHouse#49261]
Two types of failures (just one occurrence for each yet):
1) release, analyzer: https://s3.amazonaws.com/clickhouse-test-reports/49099/a67a84d4cce3ea8a24faa719af323d5f5a1630af/stateless_tests__release__analyzer_.html
2) debug: https://s3.amazonaws.com/clickhouse-test-reports/0/16eedb004f58a3c8507cf605f8e563df84dcf1e5/stateless_tests__debug__[1/3].html

**Comment by davenger** [comment]
> There is still another type of a failure.

Yep, the second is just unlucky :) There were 12 ALTERs during that test run and all of them did the same "random" alter that didn't change the query. This problem was already fixed here by not using random and didn't reproduce since then: https://github.com/ClickHouse/ClickHouse/commit/b5771016f6d53f460b635f10538df60d575dac23#diff-2203c35e66acf20b821dcdee9d55bc9a29d7754ce94974aa8fa3526222ccbf3fR63

**Comment by vdimir** [comment]
https://github.com/ClickHouse/ClickHouse/pull/56246

**Comment by rschu1ze** [comment]
Closing because https://github.com/ClickHouse/ClickHouse/issues/49261#issuecomment-1792438156 and no more failures in master in 2025.
